### PR TITLE
Created Readme.md

### PR DESCRIPTION
**Pull request by Viki-07**
Added a readme file in which I added Binary Tree DSA links.

### CODE REVIEW

**Comment by Viki-07** [comment]
@SohamRatnaparkhi

File: Readme.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Binary Tree
```
Comment (SohamRatnaparkhi) [inline Readme.md]:
``` markdown
<details>
  <summary>Binary Tree</summary>
  <a href="https://github.com/Viki-07/DSA-Course/blob/master/BinaryTree/BinaryTreeNode.java" >Binary Tree Node class</a> <br>
 <a href="https://github.com/Viki-07/DSA-Course/blob/master/BinaryTree/BInaryTreeLength.java" >Length of tree </a>
</details>
```

**Review by SohamRatnaparkhi — CHANGES_REQUESTED** [review]
Current format wont work. I have requested changes and explained the format in current issue's comment.

**Comment by SohamRatnaparkhi** [comment]
Also adding few more links will be good!

**Review by Viki-07 — COMMENTED** [review]
Reviewed and updated format

**Comment by Viki-07** [comment]
@SohamRatnaparkhi

**Comment by SohamRatnaparkhi** [comment]
Looks good to me! Will merge it.
Thankyou for contributing!

**Comment by SohamRatnaparkhi** [comment]
You created your own  README.md file but you have to make changes in already present README.md file.
You code is perfect but you need to add it after the hashmaps part in README.md file
And make a new pull request.

**Pull request merged**